### ISSUE TITLE
The back and forth with the invitation and join messages is not clear

### ISSUE THREAD

**Issue by innnkkki** [GENI-NSF/geni-portal#375]
It seems awkward that I invited some people, they get the invitation but then they have to apply for the project and then i have to approve them. It seems that even if they do use a different email address in the portal there should be a way with the invitation token to know that I have already invited them. 

If that is a lot of work, then maybe rephrase the emails that go around to better reflect the interaction. 

 _Imported from trac ticket [#375](http://trac.gpolab.bbn.com/proto-ch/ticket/375),  created by nriga on 01-16-2013 at 16:19, last modified: 07-16-2014 at 10:15_

**Comment by ahelsing** [comment]
As a project lead, how bad do you think it would be if someone handed off your invitation token to a friend, and so the wrong person was added to your project? Would you notice and correct it?


_Trac comment by ahelsing on 01-16-2013 at 16:38_

**Comment by innnkkki** [comment]
Hmm, I haven't thought of that, I have to say that it would be probably hard to notice it and most project leads will not go back to check. Maybe if the person has the same email address as the in the invitation then it can be automatically approved or else
the project lead gets a notification that says :
"Invitation sent to email A, was accepted by user with email B, do you want to let them join your project? " And they have to go through the extra step of confirming the new email. 

The main thing, I believe is for the process to be slightly different or at least different worded so it does not look like you are doing the same thing multiple times. 

_Trac comment by nriga (github user: innnkkki) on 01-16-2013 at 16:43_

**Comment by seledwards** [comment]
Right.  At a minimum that person responding to an invitation should be prompted to send an email that looks like they are some guy who wander in off the street.  It's confusing.

_Trac comment by sedwards (github user: seledwards) on 01-16-2013 at 16:46_
